### ISSUE TITLE
35754960: Provide keyboard shortcut to open context menu

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#18984]
#### Description

There was a radar for this open in 2001, believe it or not, so it has been an issue for a very long time. I don't have the old radar number, because radars are private... or they were back then anyhow.
MacOS should provide a global keyboard shortcut to open a context menu.
On Windows, it's Shift+F10. It would be optimal if the same shortcut was used, because many apps implemented it themselves, and that's the shortcut they mostly used.
Back in 2008, I was told that the philosophy is that all commands should be accessible by other menus or shortcuts, so there's "no need" for a keyboard shortcut for context menu.
If that's the case, then, why is there a context menu available to mouse (right click) and trackpad (two-finger tap) users?
At best, this missing keyboard shortcut is annoying for power keyboard users - particularly those who have grown used to having a context menu at their fingertips in Windows apps.
At worst, this missing shortcut is discriminating against those who must use a keyboard or other discrete input device to operate a computer.
If you search the web, you will find that many people have asked for this. Some have installed keyboard-mapping software just to get around this problem.
So now I would like to reopen this issue for serious consideration.

-
Product Version: All macOS since the beginning
Created: 2017-11-29T20:56:05.275910
Originated: 2017-11-29T15:49:00
Open Radar Link: http://www.openradar.me/35754960
